cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/TOOLCHAIN_GCC/TARGET_M0/irq_cm0.S

```diff
@@ -26,6 +26,9 @@
 
         .syntax  unified
 
+        // Mbed OS patch: Exclude RTE_Components.h and RTX_Config.h inclusion in .S files
```

Comment:
In commit message "cmsis: fix rtx_def inclusion in .S files for Gcc Arm", we say "they should only define macros.". Can we link to an open mbed-os issue to fix this?

Reply:
fixed